src/configure/helper/repoAnalysisHelper.ts

```diff
@@ -90,4 +101,14 @@ export class RepoAnalysisHelper {
     private GetRelativePath(workingDirectory: string, filePath: string): string{
         return path.relative(workingDirectory, filePath).split(path.sep).join('/');
     }
+
+    private getClient(serviceDefinition: IServiceUrlDefinition) {
+        let client = null;
+        if (serviceDefinition.serviceFramework == ServiceFramework.Vssf) {
+            client = new PortalExtensionRepositoryAnalysisClient(this.azureSession.credentials);
```

Comment:
We should pass the PE repo analysis url as input here as well. And remove the url from L16 in PortalExtensionRepositoryAnalysisClient.ts as it is duplicate of the PR url already specified in url helper class.